tests/functional/test_batching.py

```diff
@@ -47,11 +47,7 @@ def test_run_inference(self, resnet_8_batch_model_downloader,
 
         print("Downloaded model files:", resnet_8_batch_model_downloader)
 
-        # Starting docker with ie-serving
-        container = start_server_batch_model
-        running, logs = wait_endpoint_setup(container)
-        print("Logs from container: ", logs)
-        assert running is True, "docker container was not started successfully"
+        print("Start Ovms image: ", start_server_batch_model)

```

Comment:
Something is wrong with this line - why you print container here?

And additionally - the method which do something important should not be part of `print`